packages/build-utils/src/fs/run-user-scripts.ts

```diff
@@ -447,7 +448,7 @@ async function checkTurboSupportsCorepack(
   const turboJsonPath = path.join(rootDir, 'turbo.json');
   const turboJsonExists = await fs.pathExists(turboJsonPath);
   const turboJson = turboJsonExists
-    ? (JSON.parse(await fs.readFile(turboJsonPath, 'utf8')) as TurboJson)
+    ? (json5.parse(await fs.readFile(turboJsonPath, 'utf8')) as TurboJson)
```

Comment:
<!-- Suggestion comments propose improvements to the code. Mention specifically what should be changed and what the benefit would be.  -->
**suggestion (soft-blocking):** We should also `try-catch` and decide what to do if it fails to parse for any reason: (1) throw a better error message or (2) log the warning and do something reasonable.